lib/membrane_rtsp/response.ex

```diff
@@ -78,7 +77,7 @@ defmodule Membrane.RTSP.Response do
             {:ok, {response, rest}}
         end
 
-      _ ->
+      _else ->
```

Comment:
```suggestion
      _other ->
```
Also there is case in case a few lines above. Can we use `with` instead?
Please check other places.